playbooks/roles/epel_repositories/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+release: "{{ ansible_distribution_version.split('.')[0] }}"
```

Comment:
I think you can use `ansible_distribution_major_version` here without needing to act on the data.